server/src/main/java/org/apache/druid/client/JsonParserIterator.java

```diff
@@ -185,8 +185,15 @@ private void init()
         } else if (nextToken == JsonToken.START_OBJECT) {
           throw convertException(jp.getCodec().readValue(jp, QueryException.class));
         } else {
+          String errMsg = jp.getValueAsString();
```

Comment:
Since jp.getValueAsString() can return null 
```
Method that will try to convert value of current token to a String. JSON Strings map naturally; scalar values get converted to their textual representation. If representation can not be converted to a String value (including structured types like Objects and Arrays and null token), default value of null will be returned; no exceptions are thrown.
Since:
2.1
```
lets add a null check here.

Reply:
good catch. added checking null string and upon null value, not doing the string limitting to 192 length check.